crates/wash-lib/src/parser/mod.rs

```diff
@@ -171,7 +175,14 @@ struct RawProviderConfig {
 impl TryFrom<RawProviderConfig> for ProviderConfig {
     type Error = anyhow::Error;
 
-    fn try_from(raw_config: RawProviderConfig) -> Result<Self> {
+    fn try_from(raw_config: RawProviderConfig) -> Result<Self, Self::Error> {
+        let key_dir = if let Some(key_directory) = raw_config.key_directory {
```

Comment:
Same nit re: key_directory

Reply:
great catch. updated to a more cleaner code :)